apps/client/src/features/operator/op-event/OpEvent.tsx

```diff
@@ -0,0 +1,26 @@
+import { EventData } from 'ontime-types';
+import style from './OpEvent.module.scss';
+import { formatTime } from '../../../common/utils/time';
+
+type Props = {
+  data: any;
```

Comment:
this is an OntimeEvent
https://github.com/cpvalente/ontime/blob/master/packages/types/src/definitions/core/OntimeEvent.type.ts#L27

Generally uses of any should come with a comment